scripts/icm-injection-scripts/test-data/Containerfile

```diff
@@ -0,0 +1,3 @@
+ARG FORMAT
+FROM registry.fedoraproject.org/fedora-minimal:41
+LABEL sbom-format=$FORMAT
```

Comment:
The FORMAT argument looks unused. And if it's going to be a one-line Containerfile, we might as well create it inline in the test script

Reply:
Would you mind if we merged this PR as-is and then we can resolve this on a future commit?

Reply:
#119